src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/matchxpath/InputMatchXpathBlockComments.java

```diff
@@ -0,0 +1,15 @@
+/* // violation
```

Comment:
there should be no `// violation` please update config to avoid it.

Reply:
Done